homeassistant/components/asuswrt/router.py

```diff
@@ -319,6 +332,7 @@ async def init_sensors_coordinator(self) -> None:
             SENSORS_TYPE_COUNT: SENSORS_CONNECTED_DEVICE,
             SENSORS_TYPE_BYTES: SENSORS_BYTES,
             SENSORS_TYPE_RATES: SENSORS_RATES,
+            SENSORS_TYPE_LOAD_AVG: SENSORS_LOAD_AVG,
```

Comment:
```suggestion
            SENSORS_TYPE_COUNT: SENSORS_CONNECTED_DEVICE,
            SENSORS_TYPE_BYTES: SENSORS_BYTES,
            SENSORS_TYPE_LOAD_AVG: SENSORS_LOAD_AVG,
            SENSORS_TYPE_RATES: SENSORS_RATES,
```

Reply:
This is related to sort? So I should also invert the first 2.

Reply:
Yeah probabaly :)

Sorting = nice :)